### PR TITLE
Accept auth role arg in single task execution

### PR DESCRIPTION
**Pull request by katrogan**
# TL;DR
Accept auth role arg in single task execution. This is a required param for single task executions but currently can only be set from the user flytekit config.

## Type
 - [x] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [x] Code completed
 - [x] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
Also bump flyteidl version to pick up the fix for a non-backwards compatible field number change for auth roles.

## Tracking Issue
https://github.com/flyteorg/flyte/issues/860

## Follow-up issue
_NA_

### CODE REVIEW

**Comment by wild-endeavor** [comment]
How was this working in the past?  I know we've tested this before.

**Comment by katrogan** [comment]
you could (and still can) set the auth role in the flytekit config (or with env vars)

**Review by wild-endeavor — APPROVED** [review]
